src/help/help.go

```diff
@@ -64,6 +68,103 @@ func help(topic string) string {
 	if f, present := m[topic]; present {
 		return helpFromBuildRule(f.FuncDef)
 	}
+
+	// Check plugins
+	if pluginHelp := helpForPlugin(topic); pluginHelp != "" {
+		return pluginHelp
+	}
+
+	return ""
+}
+
+func helpForPlugin(topic string) string {
+	config, err := core.ReadDefaultConfigFiles(nil)
+	if err != nil {
+		panic("Failed to read config")
+	}
+	if _, ok := config.Plugin[topic]; ok {
+		message := fmt.Sprintf("${BOLD_BLUE}%v${RESET} is a plugin defined in the .plzconfig file.", topic)
+		if val, ok := config.Plugin[topic].ExtraValues["subrepo"]; ok {
```

Comment:
I remember us talking about having some sort of convention where we might try and find the plugin in a default place? I can't remember what we said... we can have a think about that though, as I'd like this to work even if they don't specify the subrepo.